styles/prosilver/template/event/viewtopic_body_footer_before.html

```diff
@@ -0,0 +1,6 @@
+<!-- IF not $INCLUDED_WOLFSBLVTADVANCEDPOLLSJS -->
+	<!-- INCLUDEJS js/functions.js -->
+	<!-- INCLUDEJS js/onload.js -->
+	<!-- INCLUDEJS js/scoring_topic.js -->
+    <!-- DEFINE $INCLUDED_WOLFSBLVTADVANCEDPOLLSJS = true -->
+<!-- ENDIF -->
```

Comment:
I moved it because, even though I agree with you on the fact that the code is only included once because of the IF/DEFINE, it is not "clear/clean" to include something in a loop. And it is clearer to include the code close to where it belongs (even if INCLUDEJS moves the code to the right place).
But ok, we can move it back to where it was before.